messaging/src/main/java/org/axonframework/messaging/annotations/MethodInvokingMessageHandlingMember.java

```diff
@@ -151,11 +151,10 @@ public Object handleSync(@Nonnull Message message,
                              @Nonnull ProcessingContext context,
                              T target) throws Exception {
         try {
-            return handle(message, context, target).first()
-                                                   .asCompletableFuture()
-                                                   .get()
-                                                   .message()
-                                                   .payload();
+            MessageStream.Entry<?> resultEntry = handle(message, context, target).first()
```

Comment:
I think we should mark `MethodInvokingMessageHandlingMember#handle` as `@Nullable` or force it to return at least `MessageStream.Empty`.